x-pack/plugins/ml/public/explorer/explorer.html

```diff
@@ -109,7 +109,7 @@ <h4 class="euiTitle euiTitle--small">No {{swimlaneViewByFieldName}} influencers
           <ml-annotation-table
             annotations="annotationsData"
             drill-down="true"
-            jobs="selectedJobs"
+            number-badge="false"
           />
```

Comment:
Sorry for my lack of context here but why is `number-badge` set to false here?

Reply:
That's only necessary to be enabled in Single Metric Viewer, it acts as a legend for the annotation elements shown in the chart itself. They are not visible in the jobs list or Anomaly Explorer. Since the jobs list is already React only I could just set the prop to `false` there, but the one in Anomaly Explorer needs now this additional scope attribute for the angularjs wrapper. Hope that clarifies it :).